pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -14,11 +14,22 @@ rec {
   mkDerivation =
     { name ? ""
 
-    , nativeBuildInputs ? []
-    , buildInputs ? []
+    # TODO(@Ericson2314): Stop using legacy dep attribute names
+    #                              host offset -> target offset
+    , depsBuildBuild              ? []    # -1 -> -1
```

Comment:
Perhaps a comment pointing to the discussion of these in the manual would be helpful?